agri-store-react/src/Components/Login-Signup-Inputs/LoginInputs.css

```diff
@@ -0,0 +1,15 @@
+@import "https://deploy-preview-13--agri-component.netlify.app/css/components.css";
+.Login-Form {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: silver;
+  border-radius: 0.3rem;
+  flex-direction: column;
+  padding: 3rem;
+  margin: 1rem;
+  height: auto;
+  width: 30%;
+  margin: 0 auto;
+  margin-top: 10rem;
+}
```

Comment:
Suggestion: Indentation could be better here

Reply:
Thanks for suggestions will try to follow and implement the same in next PR